servers/publikator/lib/mailchimp/updateCampaign.js

```diff
@@ -0,0 +1,54 @@
+const fetch = require('isomorphic-unfetch')
+const debug = require('debug')('publikator:lib:mailchimp:updateCampaign')
+
+const {
+  MAILCHIMP_URL,
+  MAILCHIMP_API_KEY,
+  MAILCHIMP_CAMPAIGN_CONFIGS,
+  DEFAULT_MAIL_FROM_NAME,
+  DEFAULT_MAIL_FROM_ADDRESS
+} = process.env
+
+const mailchimpCampaignConfigs = MAILCHIMP_CAMPAIGN_CONFIGS
+  ? JSON.parse(MAILCHIMP_CAMPAIGN_CONFIGS)
+  : []
+
+module.exports = async ({ campaignId, campaignConfig = {} }) => {
+  const config = {
+    ...mailchimpCampaignConfigs.find(c => c.key === campaignConfig.key),
```

Comment:
Because spreading does not return anything to merge nor does it throw an error, handling seems unnecessary.

```
> { ...[{a: 1}, {a: 2}].find(o => o.a === 3) }
{}
> { ...null }
{}
> { ...undefined }
{}
> { ...false }
{}
```